src/components/ItemCard.js

```diff
@@ -1,10 +1,10 @@
 import "./ItemCard.css";
 
-export default function ItemCard() {
+export default function ItemCard({ image }) {
   return (
     <div className="Item-card">
       <li className="Item-card">
-        <img src="image" alt="description" />
+        <img src={image} alt="description" />
         <h2 className="Item-card__name">Chocolate cake</h2>
```

Comment:
Why don't you make this dynamic? The item card should receive a prop with the name of the cake no?

Reply:
Indeed. I am planning to do this in the next PRs.